internal/service/url_service.go

```diff
@@ -21,23 +26,76 @@ func NewURLService(repo repository.URLRepository) URLService {
 }
 
 func (s urlService) CreateShortURL(req models.ShortenRequest) (string, error) {
-	//TODO implement me
-	panic("implement me")
+	short := shortener.GeneratorShortURL()
+	log.Println(short)
+	_, err := s.repo.GetURL(short)
+	if err != nil {
+		return "", fmt.Errorf("short url already exists %s ", short)
```

Comment:
why in error you write generated short (from line 29 (short := shortener.GeneratorShortURL()))
and better write error: fmt.Errorf("create short url, get url err: %s ", err), because there can be other errors, not only already exists

Reply:
fix format errors everywhere